### PR TITLE
Proserve restructure with Matts changes

### PR DESCRIPTION
**Pull request by olepinard**

### CODE REVIEW

**Comment by mattballard-planet** [comment]
I think it looks good! The only comment I have is that I am sort of unsure about whether I made the recommendation to bring the APIs up a level. I like to that they are more prominent, but I also see why they would be grouped into an `apis` folder.

Either way - it looks great from my perspective! I'd be curious if @emma-steuer or @karrabatcheller have an opinion on the folder structure?

**Comment by olepinard** [comment]
I think having the APIs moved up makes more sense now that there are less other folders and everything feels more concise. APIs are the main reason users seek out these notebooks so I think that makes sense

File: README.md [hunk 1]
```diff
@@ -116,8 +114,25 @@ http://localhost:8888/?token=<UNIQUE-TOKEN>
...
+### jupyter-notebooks
+
+#### exploring_planet_data: Working with our various image products, how to use the udm mask or deliver imagery to our GEE integration
+
+* [How to use the Data and Orders API to create analysis ready data](https://github.com/planetlabs/notebooks/tree/proserve_restructure/jupyter-notebooks/exploring_planet_data/analysis-ready-data)
+* [Converting Raster Results to Vector Features](https://github.com/planetlabs/notebooks/blob/master/jupyter-notebooks/analytics-snippets/README.md)
+* [Introduction to Cloud Native Geospatial Tools](https://github.com/planetlabs/notebooks/tree/master/jupyter-notebooks/cloud-native-geospatial)
+* [Comparing Planet Scope with Landsat 8]()
+* [Deliver data to GEE](https://github.com/planetlabs/notebooks/tree/master/jupyter-notebooks/cloud-native-geospatial)
+* [Start working with sattelite imagery in Python](https://github.com/planetlabs/notebooks/tree/master/jupyter-notebooks/cloud-native-geospatial)
```
Comment (karrabatcheller) [inline README.md:128]: spelling error -  sattelite --> satellite

File: README.md [hunk 2]
```diff
@@ -116,8 +114,25 @@ http://localhost:8888/?token=<UNIQUE-TOKEN>
...
+## Repository Organization
+
+### jupyter-notebooks
+
+#### exploring_planet_data: Working with our various image products, how to use the udm mask or deliver imagery to our GEE integration
+
+* [How to use the Data and Orders API to create analysis ready data](https://github.com/planetlabs/notebooks/tree/proserve_restructure/jupyter-notebooks/exploring_planet_data/analysis-ready-data)
+* [Converting Raster Results to Vector Features](https://github.com/planetlabs/notebooks/blob/master/jupyter-notebooks/analytics-snippets/README.md)
+* [Introduction to Cloud Native Geospatial Tools](https://github.com/planetlabs/notebooks/tree/master/jupyter-notebooks/cloud-native-geospatial)
+* [Comparing Planet Scope with Landsat 8]()
```
Comment (karrabatcheller) [inline README.md:126]: It looks like is this missing a link

**Review by karrabatcheller — APPROVED** [review]

**Pull request merged**